docs/real-world-examples/ccfraud.md

```diff
@@ -31,6 +31,30 @@ In the next section, we will run a job in the AzureML workspace that will unpack
 
 Kaggle required username and a developer key, so we will first store safely those credentials in the workspace keyvault.
 
+### Using Azure CLI
+
+1. Get the signed-in user's object ID. 
+```bash
+az ad signed-in-user show | jq ".id"
+```
+2. Grant permissions to *list, set & delete* secrets by setting a keyvault policy. 
```

Comment:
rewrite:

Create a new keyvault policy for yourself, and grant permissions to *list, set & delete* secrets.